addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -34,6 +34,18 @@
                             />
               </div>
             </field>
+            <page name="credentials" position="attributes">
+                <attribute name="invisible" add="custom_mode == 'wire_transfer'" separator=" or "/>
```

Comment:
If only this module has been installed, the invisible attribute is not defined, so there's nothing before the "or" and an error appears (although it worked on the runbot since all modules were installed).

Reply:
Fair point, I had in mind to default to `False` but it's indeed more logical to rely on the separator for that. I'll just swap the attributes then.